input/pagecontent/5_glossary.md

```diff
@@ -2,11 +2,14 @@
 {:.grid}
 Term | Description
 ---- | ---
-**Anchor Context** | a context that is used as a container for shared content, typical anchor contexts are FHIR resources such as `ImagingStudy` and `DiagnosticReport`
+
+**Anchor Context** | a context that is used as a container for shared content, typical anchor contexts are FHIR resources such as Patient, Encounter, ImagingStudy, and DiagnosticReport
+**Anchor resource** | a FHIR resource that is used as a container for shared content, typical anchor resource are resources such as Patient, Encounter, ImagingStudy, and DiagnosticReport
```

Comment:
Why is there an Anchor resource?  I thought throughout the spec we were using Anchor Context and it was fine.  For what is Anchor resource used?